### PR TITLE
#27 - faq page

### LINKED ISSUE #27: FAQ page
FAQ items should be visible for guests on `/faq` page.

This is related to #14  and should be done after implementation of this task.

### PR DESCRIPTION
**Pull request by kamilpiech97**
Should close #27.

Preview:

image

### CODE REVIEW

File: resources/js/Pages/Public/Faq.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="relative isolate bg-white">
+      <BackgroundGrid />
+      <div class="py-24 sm:py-32">
+        <div class="mx-auto max-w-7xl px-6 lg:px-8">
+          <SectionHeader>
+            <template #header>
+              FAQ
+            </template>
+            <template #subheader>
+              Pytania i odpowiedzi na najczęściej zadawane pytania
```
Comment (EwelinaSkrzypacz) [inline resources/js/Pages/Public/Faq.vue:34]:
```suggestion
              Często zadawane pytania i odpowiedzi na nie
```

**Review by EwelinaSkrzypacz — COMMENTED** [review]
I'm not sure if we need this +/- icon to reveal/hide answers. Also the answer is not on full width

image

**Comment by krzysztofrewak** [comment]
I would like to remove this collapsing feature. It's not necessary, I think.

**Comment by kamilpiech97** [comment]
> I would like to remove this collapsing feature. It's not necessary, I think.

Removed

**Comment by kamilpiech97** [comment]
> I'm not sure if we need this +/- icon to reveal/hide answers. Also the answer is not on full width
> 
> image

Updated. Image in description updated also :D

**Review by EwelinaSkrzypacz — DISMISSED** [review]

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**